### PR TITLE
Add Garavito keymap to lily58

### PR DESCRIPTION
**Pull request by EGaravitoM**
## Description

Add Garavito keymap to Lily58

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [ ] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/lily58/keymaps/garavito/config.h [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+
+#pragma once
+
+//#define USE_MATRIX_I2C
+
+/* Select hand configuration */
+
+#define MASTER_LEFT
+// #define MASTER_RIGHT
+// #define EE_HANDS
```
Comment (dunk2k) [inline keyboards/lily58/keymaps/garavito/config.h:29]: not required:
1. MASTER_LEFT is default handedness settings for split keyboards
2. doesn't override handedness settings from directories above, as there are none.
```suggestion
```

File: keyboards/lily58/keymaps/garavito/rules.mk [hunk 2]
```diff
@@ -0,0 +1,6 @@
+VIA_ENABLE = yes
+OLED_ENABLE = yes
+OLED_DRIVER = SSD1306
```
Comment (dunk2k) [inline keyboards/lily58/keymaps/garavito/rules.mk:3]: already defined in `rules.mk` of directories above
```suggestion
```

**Review by dunk2k — CHANGES_REQUESTED** [review]

File: keyboards/lily58/keymaps/garavito/keymap.c [hunk 3]
```diff
@@ -0,0 +1,287 @@
...
+oled_rotation_t oled_init_user(oled_rotation_t rotation) {
+  if (!is_keyboard_master())
+    return OLED_ROTATION_180;  // flips the display 180 degrees if offhand
+  return rotation;
+}
+
+// layer_state_t layer_state_set_user(layer_state_t state) {
+//     state = update_tri_layer_state(state, _RAISE, _LOWER, _ADJUST);
+//     return state;
+// }
```
Comment (waffle87) [inline keyboards/lily58/keymaps/garavito/keymap.c:157]:
```suggestion
```

File: keyboards/lily58/keymaps/garavito/config.h [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+You should have received a copy of the GNU General Public License
+along with this program.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+#pragma once
+
+//#define USE_MATRIX_I2C
+
+/* Select hand configuration */
+
```
Comment (waffle87) [inline keyboards/lily58/keymaps/garavito/config.h:26]:
```suggestion
```

**Review by waffle87 — CHANGES_REQUESTED** [review]

**Comment by zvecr** [comment]
Unfortunately since this PRs creation, QMK has [changed its stance](https://docs.qmk.fm/#/ChangeLog/20230827?id=notable-changes) on accepting user keymaps to the repository.

> qmk/qmk_firmware is no longer accepting PRs for keymaps other than for manufacturer-supported keymaps. User keymap workflow has been documented [here](https://docs.qmk.fm/#/newbs) for several years. This change is to progressively reduce the maintenance burden on the project, and to allow us to focus on the core features of QMK.

In the near future we will be providing a more user focused workflow for keymaps, but till then the suggestion is to keep changes in your own fork.

While we appreciate the time that was taken on this PR, hopefully you understand we are closing it, so that QMK can improve in the long term.

**Pull request closed without merging**